lib/msf/core/data_store.rb

```diff
@@ -169,29 +169,29 @@ def to_s(delim = ' ')
   def to_h
     datastore_hash = {}
     self.keys.each do |k|
-      datastore_hash[k.to_s] = self[k].to_s
+      datastore_hash[k.to_s.dup.force_encoding('UTF-8')] = self[k].to_s.dup.force_encoding('UTF-8')
```

Comment:
I believe we can revert this line as only the other method is called as part of external modules? 👀